_includes/releases/v22.1/v22.1.0-beta.3.md

```diff
@@ -0,0 +1,78 @@
+## v22.1.0-beta.3
+
+Release Date: April 18, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-3-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Unified the syntax for defining the behavior of initial scans on [changefeeds](../v22.1/create-changefeed.html) by extending the `initial_scan` option to accept three possible values: `'yes|no|only'`. [#79471][#79471]
+- Changefeed messages for tables with multiple [column families](../v22.1/column-families.html) using the `split_column_families` option or the [`CREATE CHANGEFEED for TABLE bar FAMILY f_1`](../v22.1/create-changefeed.html) syntax will append the family name to the table name in the default topic. For example, `CREATE CHANGEFEED FOR TABLE foo, TABLE bar FAMILY f_1` will emit to the topics `foo` and `bar.f_1`.  In cloud storage sinks such as s3, the filename will include the family separated by a `+` (e.g., `bar+f_1`). A changefeed created with the `split_column_families` option will create new topics if new families are added to a watched table that already had multiple column families. [#79448][#79448]
+- The `full_table_name` option is now supported for all [changefeed](../v22.1/change-data-capture-overview.html) sinks. [#79448][#79448]
+- `LATEST` files are no longer overwritten and are now versioned and written in the `/metadata/latest` directory for non-mixed clusters. [#79553][#79553]
+- Previously, the [`ALTER CHANGEFEED`](../v22.1/alter-changefeed.html) statement would not work with changefeeds that use fully qualified names in their [`CREATE CHANGEFEED`](../v22.1/create-changefeed.html) statement. This is now fixed by ensuring that each existing target is added with its fully qualified name so that it can be resolved in validation checks. Every changefeed will now display the fully qualified name of every target in the [`SHOW CHANGEFEED JOB`](../v22.1/show-jobs.html) query. [#79745][#79745]
+- Added a `changefeed.backfill.scan_request_size` setting to control scan request size during backfill. [#79710][#79710]
+
+<h3 id="v22-1-0-beta-3-sql-language-changes">SQL language changes</h3>
+
+- CockroachDB now ensures the user passes the same number of locality-aware URIs for the full [backup](../v22.1/take-full-and-incremental-backups.html) destination as the `incremental_location` parameter (i.e., `BACKUP INTO LATEST IN ($1, $2, $3) WITH incremental_location = ($4, $5, $6)`). [#79600][#79600]
+- `EXPLAIN (DDL)`, when invoked on statements supported by the declarative schema changer, prints a plan of what the schema changer will do. This can be useful for anticipating the complexity of a schema change (e.g., anything involving backfill or validation operations might be slow to run) and for troubleshooting. `EXPLAIN (DDL, VERBOSE)` produces a more detailed plan. [#79780][#79780]
```

Comment:
```suggestion
- `EXPLAIN (DDL)`, when invoked on statements supported by the declarative schema changer, prints a plan of what the schema changer will do. This can be useful for anticipating the complexity of a schema change (for example, anything involving backfill or validation operations might be slow to run) and for troubleshooting. `EXPLAIN (DDL, VERBOSE)` produces a more detailed plan. [#79780][#79780]
```